src/box/lua/call.c

```diff
@@ -1269,6 +1269,15 @@ lbox_func_new_or_delete(struct trigger *trigger, void *event)
 	return 0;
 }
 
+static int
+lbox_rt_lua_call_access_reset(struct lua_State *L)
+{
+	if (lua_gettop(L) != 0)
+		return luaL_error(L, "internal.clear_lua_call bad arguments.");
+	rt_lua_call_access_reset();
+	return 0;
+}
```

Comment:
It is better to move it into the patch, where the corresponding grant function is added.

Reply:
Done.